intellij-plugin-structure/structure-intellij/src/main/java/com/jetbrains/plugin/structure/intellij/version/ProductReleaseVersion.kt

```diff
@@ -0,0 +1,36 @@
+package com.jetbrains.plugin.structure.intellij.version
+
+
+private const val RELEASE_VERSION_TAG_NAME = "release-version"
+/**
+ * Type-safe `release-version` of paid plugins.
+ *
+ * Relevant docs:
+ * - See
+ * [Plugin Configuration file](https://plugins.jetbrains.com/docs/intellij/plugin-configuration-file.html#idea-plugin__product-descriptor)
+ * at IntelliJ Platform Plugin SDK
+ * - [JetBrains Marketplace](https://plugins.jetbrains.com/docs/marketplace/add-required-parameters.html) guidelines
+ */
+data class ProductReleaseVersion(val value: Int) {
```

Comment:
I understand it handles versions with a single-digit minor version.

Are we sure there are no plugins releasing a major version more than 9 times a year? For example, `202412`. I think it is worth checking if it was not.